channel/consensus_channel/consensus_channel.go

```diff
@@ -575,10 +586,11 @@ func (sp *SignedProposal) Clone() SignedProposal {
 }
 
 // Add encodes a proposal to add a guarantee to a ConsensusChannel.
-//
-// The balance of the guarantee is to be deducted from left.
 type Add struct {
 	Guarantee
+	// LeftDeposit is the portion of the Add's amount that will be depositied by the left participant.
```

Comment:
```suggestion
	// LeftDeposit is the portion of the Add's amount that will be deposited by the left participant.
```

Reply:
I think I prefer `deducted from` rather than `deposited by`, since we tend to use deposit to refer to an on chain deposit.